src/tasks.ts

```diff
@@ -43,14 +48,26 @@ export type RetriableTask<E, T> = E extends TransientError
 /**
  * Wraps a RetriableTask with a number of retries
  */
+// tslint:disable-next-line: cognitive-complexity
 export function withRetries<E, T>(
   maxRetries: number,
-  backoff: (count: number) => Millisecond
+  backoff: (count: number) => Millisecond,
+  controllerRetry?: Promise<Response>
 ): (
   _: RetriableTask<E, T>,
   shouldAbort?: Promise<boolean>
 ) => TaskEither<E | MaxRetries | RetryAborted, T> {
   return (task, shouldAbort = Promise.resolve(false)) => {
+    // tslint:disable-next-line: no-let
```

Comment:
this is basically a duplication of the logic implemented by passing `shouldAbort`, the fix should be done by passing `shouldAbort` [here](https://github.com/teamdigitale/io-ts-commons/blob/master/src/fetch.ts#L126)